src/components/bcros/businessDetails/Links.vue

```diff
@@ -243,7 +262,7 @@ const contacts = getContactInfo('registries')
     </BcrosDialog>
 
     <!-- Staff Comments -->
-    <div v-if="hasRoleStaff && currentBusiness && !isDisableNonBenCorps()">
```

Comment:
Don't we want to disable staff comments for non-Ben corps (that should be managed in COLIN instead)?

Reply:
It's not enabled in Filings UI. LMK if this is a change in requirements.

![image](https://github.com/user-attachments/assets/a979a027-4080-4ea2-8d92-7bbcca84c435)

Reply:
Accidently chucked this commit when adding a new one, but I didn't change that line